common/djangoapps/util/tests/test_help_context_processor.py

```diff
@@ -0,0 +1,81 @@
+"""
+Tests for help_context_processor.py
+"""
+
+import ConfigParser
+from mock import patch
+
+from django.conf import settings
+from django.test import TestCase
+
+from util.help_context_processor import common_doc_url
+
+
+CONFIG_FILE = open(settings.REPO_ROOT / "docs" / "lms_config.ini")
+CONFIG = ConfigParser.ConfigParser()
+CONFIG.readfp(CONFIG_FILE)
+
+
+class HelpContextProcessorTest(TestCase):
```

Comment:
No tests existed for HelpContextProcessor before. I mainly just stuck to testing the lines of code that were changed.